airbyte-integrations/connectors/source-zendesk-sell/source_zendesk_sell/spec.yaml

```diff
@@ -0,0 +1,24 @@
+documentationUrl: https://developer.zendesk.com/api-reference/sales-crm/introduction/#get-up-to-speed-with-the-sell-api
```

Comment:
This is Airbyte's documentation to Zendesk Sell Source Page. Please updaate.

Reply:
done